apps/core/client/fragments/prices.ts

```diff
@@ -0,0 +1,34 @@
+import { graphql } from '../graphql';
+
+export const PRICES_FRAGMENT = graphql(`
```

Comment:
Moved this to a "global" fragment as it's currently being exported from `get-product` and being used in multiple places.